slynk/slynk-backend.lisp

```diff
@@ -196,7 +196,7 @@ Backends implement these functions using DEFIMPLEMENTATION."
        (if (member ',sym *interface-functions*)
            (setq *unimplemented-interfaces*
                  (remove ',sym *unimplemented-interfaces*))
-           (warn "DEFIMPLEMENTATION of undefined interface (~S)" ',sym))
+           (warn "DEFIMPLEMENTATION of undefined interface (~S)" ',name))
```

Comment:
I think this makes sense.  Can you give a very summarized before&after of what this change does to the logs?

Reply:
The DEFIMPLEMENTATIONs that I removed from clasp.lisp triggered this warning and they read:

`WARNING: DEFIMPLEMENTATION of undefined interface (NIL)`

, which wasn't useful.  After this change you will see warnings like:

`WARNING: DEFIMPLEMENTATION of undefined interface (DUMMY-IMPLEMENTATION)`

I tested this on sbcl and in clasp.